src/evented-tokenizer.ts

```diff
@@ -113,21 +114,31 @@ export default class EventedTokenizer {
     this.delegate.appendToTagName(char);
   }
 
+  private isIgnoredEndTag(): boolean {
+    let tag = this.tagNameBuffer.toLowerCase();
+
+    return (tag === 'title' && this.input.substr(this.index, 8) !== '</title>') ||
```

Comment:
Lets avoid [String.prototype.substr](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) here (since its "deprecated"), maybe use `this.input.substring(this.index, this.index + 8)`?

Reply:
Good catch, my bad! Fixed.